src/components/lib/Confirming.js

```diff
@@ -0,0 +1,56 @@
+import React from "react"
+import PropTypes from "prop-types"
+
+import Description from "./Description"
+import StatusIndicator from "../svgs/StatusIndicator"
+
+const Confirming = ({
+  className,
+  stepStatus,
+  error,
+  requiredConfirmations,
+  confirmations,
+  children,
+  extraMessage,
```

Comment:
How about losing the `extraMessage` and moving it into the `children` piece? I know it changes page structure a little, but the current page structures are not long for the world anyway, so I figure maybe let's simplify our components while we're moving along?

Reply:
That feels right! I wasn't really happy about making the `extraMessage` component but was hesitant to alter the page structure. I'll move it into the children!

Reply:
Removed!